### ISSUE TITLE
[BUG] PYTHON Generated Client unable to parse response including array: [SCHEMA_OBJECT]

### ISSUE THREAD

**Issue by nmartins** [OpenAPITools/openapi-generator#10564]
##### Description

PYTHON generated client is crashing, unable to parse response containing an array[SCHEMA_OBJECT]

this is the stack trace:
Traceback (most recent call last):
  File "/gen-cli-python/start-python-query-example.py", line 47, in <module>
    pprint(api_instance.group_query_external_id(external_id))
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 53, in pprint
    printer.pprint(object)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 148, in pprint
    self._format(object, self._stream, 0, 0, {}, 0)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 170, in _format
    rep = self._repr(object, context, level)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 431, in _repr
    repr, readable, recursive = self.format(object, context.copy(),
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 444, in format
    return _safe_repr(object, context, maxlevels, level, self._sort_dicts)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pprint.py", line 596, in _safe_repr
    rep = repr(object)
  File "/gen-cli-python/openapi_client/model_utils.py", line 176, in __repr__
    return self.to_str()
  File "/gen-cli-python/openapi_client/model_utils.py", line 517, in to_str
    return pprint.pformat(self.to_dict())
  File "/gen-cli-python/openapi_client/model_utils.py", line 513, in to_dict
    return model_to_dict(self, serialize=False)
  File "/gen-cli-python/openapi_client/model_utils.py", line 1662, in model_to_dict
    res.append(model_to_dict(v, serialize=serialize))
  File "/gen-cli-python/openapi_client/model_utils.py", line 1662, in model_to_dict
    res.append(model_to_dict(v, serialize=serialize))
  File "/gen-cli-python/openapi_client/model_utils.py", line 1634, in model_to_dict
    if model_instance._composed_schemas:
AttributeError: 'dict' object has no attribute '_composed_schemas'

##### openapi-generator version
5.2.1

##### OpenAPI declaration file content or url

[specs_and_example.zip](https://github.com/OpenAPITools/openapi-generator/files/7312056/specs_and_example.zip)

This is the Json Response example

```json
{
    "adminCount": 0,
    "adminCursor": 0,
    "currentStatusTransitionTime": "2021-10-08T13:39:58.000000Z",
    "deviceCount": 0,
    "externalId": "group123456789",
    "groupMemberCount": 0,
    "groupMemberCursor": 0,
    "lastActivityTime": "2021-10-08T13:39:58.000000Z",
    "maxSubscriberCount": 4294967295,
    "maxUserCount": 4294967295,
    "mtx_container_name": "MtxResponseGroup",
    "mtx_ext_ver": 1,
    "mtx_result_code": 0,
    "mtx_result_text": "OK",
    "mtx_result_type": "get",
    "mtx_sys_ver": 5240,
    "name": "This is the first group",
    "notificationPreference": 1,
    "objectId": "0-1-5-6",
    "relatedUserArray": [
        {
            "externalId": "user123456789",
            "mtx_container_name": "MtxRelatedUserObject",
            "mtx_ext_ver": 1,
            "mtx_sys_ver": 5240,
            "objectId": "0-1-5-1",
            "roleInfoArray": [
                {
                    "externalId": "Admin",
                    "mtx_container_name": "MtxPricingRoleInfo",
                    "mtx_ext_ver": 1,
                    "mtx_sys_ver": 5240,
                    "name": "Admin",
                    "pricingId": 2
                }
            ]
        }
    ],
    "relatedUserCursor": 0,
    "result": 0,
    "resultText": "OK",
    "routeId": 1,
    "status": 1,
    "statusDescription": "Active",
    "subscriberCount": 0,
    "subscriberMemberCount": 0,
    "subscriberMemberCursor": 0,
    "userCount": 1
}
```
It's failing to parse relatedUserArray into MtxRelatedUserObject

##### Generation Detail

java -jar openapi-generator-cli-5.2.1.jar generate -i openapi3.json --verbose -g python -o gen-cli-python

##### Steps to reproduce
1. generate the client using openapi3.json file in attach
2. using command: java -jar openapi-generator-cli-5.2.1.jar generate -i openapi3.json --verbose -g python -o gen-cli-python
3. run python3 gen-cli-python/setup.py install
4. copy start-python-query-example.py in attach into gen-cli-python/
5. run python example in attach, against a server responding the json response example provided above with following command: "python3 start-python-query-example.py", and it'll output the stack trace provided above.

**Comment by SamyMe** [comment]
Hello, Did you make it work ? I'm running into the same problem.

**Comment by flashgorman-chorus** [comment]
Experiencing the same thing.  Please advise on the status of this bug.

**Comment by mickelsonmichael** [comment]
More support behind this issue, I'm also experiencing it

**Comment by TomTom101** [comment]
Same here

**Comment by MasterDuke17** [comment]
I believe this was fixed in 5.4 with https://github.com/OpenAPITools/openapi-generator/pull/11234.

**Comment by spacether** [comment]
@nmartins your spec has an issue
- you define type object adjacent to $ref definition
That is not allowed per https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#fixed-fields-19
`This object (schema containing $ref) cannot be extended with additional properties and any properties added SHALL be ignored.`

Using `python` v6.2.0 your spec just successfully generated a client
